### ISSUE TITLE
NPE with RepoCloner

### ISSUE THREAD

**Issue by MagielBruntink** [fasten-project/fasten#211]
## Describe the bug

NPE while running RepoCloner as plugin, seems to originate from:
`https://github.com/fasten-project/fasten/blob/d0860d060be4074740402488be13213555085600/analyzer/repo-cloner-plugin/src/main/java/eu/fasten/analyzer/repoclonerplugin/RepoClonerPlugin.java#L140`

## To Reproduce
1. Clone https://github.com/fasten-project/fasten-docker-deployment
2. cd into fasten-docker-deployment
3. In docker-compose.yml on line 40 change e89ba04c into latest
4. docker-compose pull && docker-compose up
5. Wait a bit until all containers are up
6. cat test-resources/mvn.coords.txt | head -1 | docker-compose exec -T kafka kafka-console-producer --broker-list kafka:19092 --topic fasten.mvn.test --property 'parse.key=true' --property 'key.separator=|'
7. Observe NPE in log of the repo-cloner container, or on the console.

## Expected behavior
No NPE :-)

## Logs
```
[2021-01-30 09:43:34,081] [DEBUG] [main] [o.e.j.u.FS] - readpipe [git, --version],/usr/bin
[2021-01-30 09:43:34,088] [DEBUG] [main] [o.e.j.u.FS] - readpipe may return 'git version 2.20.1'
[2021-01-30 09:43:34,089] [DEBUG] [main] [o.e.j.u.FS] - remaining output:
[2021-01-30 09:43:34,089] [DEBUG] [main] [o.e.j.u.FS] - readpipe [git, config, --system, --edit],/usr/bin
[2021-01-30 09:43:34,091] [DEBUG] [main] [o.e.j.u.FS] - readpipe may return '/etc/gitconfig'
[2021-01-30 09:43:34,091] [DEBUG] [main] [o.e.j.u.FS] - remaining output:
[2021-01-30 09:43:34,735] [ERROR] [main] [e.f.s.p.k.FastenKafkaPlugin] - Error occurred while processing call graphs
java.lang.NullPointerException: null
at eu.fasten.analyzer.repoclonerplugin.RepoClonerPlugin$RepoCloner.consume(RepoClonerPlugin.java:140)
at eu.fasten.server.plugins.kafka.FastenKafkaPlugin.processRecord(FastenKafkaPlugin.java:235)
at eu.fasten.server.plugins.kafka.FastenKafkaPlugin.handleConsuming(FastenKafkaPlugin.java:179)
at eu.fasten.server.plugins.kafka.FastenKafkaPlugin.run(FastenKafkaPlugin.java:142)
at eu.fasten.server.plugins.kafka.FastenKafkaPlugin.start(FastenKafkaPlugin.java:160)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at eu.fasten.server.FastenServer.run(FastenServer.java:205)
at picocli.CommandLine.executeUserObject(CommandLine.java:1919)
at picocli.CommandLine.access$1100(CommandLine.java:145)
at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
at picocli.CommandLine.execute(CommandLine.java:2058)
at eu.fasten.server.FastenServer.main(FastenServer.java:342)
[2021-01-30 09:43:34,735] [INFO ] [main] [o.a.k.c.c.i.AbstractCoordinator] - [Consumer clientId=eu.fasten.analyzer.repoclonerplugin.RepoClonerPlugin.RepoCloner_client, groupId=eu.fasten.analyzer.repoclonerplugin.RepoClonerPlugin.RepoCloner] Member eu.fasten.analyzer.repoclonerplugin.RepoClonerPlugin.RepoCloner_client-2692f9dd-0ae5-4beb-8262-edb1225bb6d1 sending LeaveGroup request to coordinator kafka:19092 (id: 2147483646 rack: null)
[2021-01-30 09:43:34,741] [INFO ] [main] [e.f.s.p.k.FastenKafkaPlugin] - Plugin Repo Cloner Plugin stopped
```

## Environment (please complete the following information):
 - OS: Docker/Linux
 - Java version: 11

**Comment by MihhailSokolov** [comment]
Should be resolved by 6e99af7

**Comment by MagielBruntink** [comment]
Yep, fixed, great!
